packages/shortest/src/ai/client.ts

```diff
@@ -42,6 +42,9 @@ export class AIClient {
     tokenUsage: { input: number; output: number };
     pendingCache: any;
   }> {
+    // temp cache store
+    const pendingCache: PendingCache = {};
```

Comment:
If a comments is needed to document it, then should use a more appropriate name that self documents.

```suggestion
    const tempCache: TempCache = {};
```